gammapy/stats/tests/test_variability.py

```diff
@@ -143,3 +144,51 @@ def test_lightcurve_flux_doubling():
         [2271.34711286, 21743.98603654] * u.s,
     )
     assert_allclose(dtime_err, [425.92375713, 242.80234065] * u.s)
+
+
+def test_structure_function():
+    flux = np.array(
+        [
+            [1e-11, 4e-12],
+            [3e-11, 2.5e-12],
+            [1e-11, 1e-12],
+            [0.8e-11, 0.8e-12],
+            [1e-11, 1e-12],
+        ]
+    )
```

Comment:
the flux units are missing.... the same for their errors

Reply:
The units on flux and flux errors are missing in all tests of similar type. They are not necessary in a sense (all these functions could work on an dimensional normalization). Should this be addressed ina  separate PR?

Reply:
If it works for both cases, I think that there is no need of an other PR... If it works with units;)